package.json

```diff
@@ -52,7 +52,6 @@
     "watch": "rollup -w -c build/rollup-config.js",
     "uglify": "uglifyjs dist/leaflet-src.js -c -m -o dist/leaflet.js --source-map filename=dist/leaflet.js.map --source-map content=dist/leaflet-src.js.map --source-map url=leaflet.js.map --comments",
     "bundlemon": "bundlemon --subProject js --defaultCompression none && bundlemon --subProject js-gzip --defaultCompression gzip",
-    "serve": "cd docs && bundle exec jekyll serve",
```

Comment:
Any reason to remove this, or does it stop working after the PR updates?

Reply:
I would like to keep it

Reply:
Removed it since the docs pretty much live on their own and don't need NPM. This way the docs stay very pure on the Ruby side of things.